tests/test_database_types.py

```diff
@@ -595,7 +599,10 @@ def test_types(self, source_db, target_db, source_type, target_type, type_catego
         download_duration = time.time() - start
         expected = []
         self.assertEqual(expected, diff)
-        self.assertEqual(len(sample_values), differ.stats.get("rows_downloaded", 0))
+        if type_category == "uuid":
+            pass  # UUIDs aren't serial, so they mess with the first max_rows estimation.
```

Comment:
We should still be able to detect how many rows were downloaded though... no?

Reply:
The problem is we're calculating `max_rows` as the distance between min and max.. but with UUIDs it's a huge number. Only in seconds iteration we get a real count.

This is something we might want to fix in the future, but I think it only interferes with the tests, not a realistic scenario.

Reply:
But `rows_downloaded` is _after_ we do `get_values` and based on `len(rows1)`?

Reply:
Ooooh, I understand now, it's because it doesn't hit the `bisection_threshold`?

Reply:
Could you make the `bisection_threshold` just infinite then? These tests to check that equality works in Python too I think are still valuable.